server/db/migrate/20190327073533_create_blocks.rb

```diff
@@ -0,0 +1,33 @@
+class CreateBlocks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :blocks do |t|
+      t.binary :cellbase_id
+      t.binary :difficulty
+      t.binary :block_hash
+      t.integer :number
+      t.binary :parent_hash
+      t.jsonb :seal
+      t.integer :timestamp
+      t.binary :txs_commit
+      t.binary :txs_proposal
+      t.integer :uncles_count
+      t.binary :uncles_hash
+      t.string :uncle_block_hashes
+      t.integer :version
+      t.binary :proposal_transactions
```

Comment:
Only one or an array ?

Reply:
it is an array, but I want pack them into a binary sequence

Reply:
Ok